platform/android/api/java_class_wrapper.h

```diff
@@ -134,23 +134,23 @@ class JavaClass : public Reference {
 				likelihood = 0.5;
 				break;
 			case ARG_ARRAY_BIT | ARG_TYPE_SHORT:
-				r_type = Variant::PACKED_INT_ARRAY;
+				r_type = Variant::PACKED_INT32_ARRAY;
 				likelihood = 0.3;
 				break;
 			case ARG_ARRAY_BIT | ARG_TYPE_INT:
-				r_type = Variant::PACKED_INT_ARRAY;
+				r_type = Variant::PACKED_INT32_ARRAY;
 				likelihood = 1.0;
 				break;
 			case ARG_ARRAY_BIT | ARG_TYPE_LONG:
-				r_type = Variant::PACKED_INT_ARRAY;
+				r_type = Variant::PACKED_INT32_ARRAY;
 				likelihood = 0.5;
 				break;
 			case ARG_ARRAY_BIT | ARG_TYPE_FLOAT:
-				r_type = Variant::PACKED_REAL_ARRAY;
+				r_type = Variant::PACKED_FLOAT32_ARRAY;
 				likelihood = 1.0;
 				break;
 			case ARG_ARRAY_BIT | ARG_TYPE_DOUBLE:
-				r_type = Variant::PACKED_REAL_ARRAY;
+				r_type = Variant::PACKED_FLOAT32_ARRAY;
```

Comment:
I'm not familiar with the Java glue but I guess this should be 64-bit actually?

Reply:
In Java, the convention is to always use doubles unless there is a specific reason not to. So yes, I think this should definitely be 64-bit.